rules/no-abusive-eslint-disable.js

```diff
@@ -14,15 +14,14 @@ const create = context => ({
 				!res[2] // But it did not specify any rules
 			) {
 				context.report({
-					// Can't set it at the given location as the warning
-					// will be ignored due to the disable comment
```

Comment:
Can you explain how the above does not apply anymore with your changes?

Reply:
Eslint use the line and column position to determine whether a report will be displayed.
When the column is `-1` it is displayed (Will ignore the deactivation via `eslint-disable-line`, `eslint-disable-next-line` ...).

![image](https://user-images.githubusercontent.com/5731176/59555242-e52f1080-8f85-11e9-8605-16454562a377.png)
**Note**: `-1` becomes `0` in issue list